doc/dev/index.rst

```diff
@@ -723,15 +723,12 @@ command to complete successfully on x86_64 (other architectures may have
 different constraints). Depending on your hardware, it can take from 20
 minutes to three hours to complete, but it's worth the wait.
 
-Future sections
----------------
+Caveats
+-------
 
-* Principles of `make check`_ tests
-* Where to find test results
-* How to interpret test results
-* Find the corresponding source code
-* Writing `make check`_ tests
-* Caveats
+1. The tests in `make check`_ are not linked against libtcmalloc by default,
```

Comment:
better off putting

> Unlike ceph daemons and ceph-fuse, the unit tests are always linked against the default glibc allocator.

otherwise we need to enumerate all supported allocators (for instance, jemalloc, tcmalloc_minimal) here.